extensions/obviousAlexC/SensingPlus.js

```diff
@@ -128,6 +134,19 @@
         MATRIX: "matrix",
         NOTE: "note",
       },
+      Cast: {

```

Comment:
FYI: this is fine for now, but with the added ability to properly run extensions from files, everything like this is probably going to be removed soon

Reply:
> FYI: this is fine for now, but with the added ability to properly run extensions from files, everything like this is probably going to be removed soon

Yeah, the only reason I did add that was to test on other devices since I'm only at home for a small portion of the day.